packages/instantsearch.js/src/components/Pagination/Pagination.tsx

```diff
@@ -81,7 +81,7 @@ function Pagination(props: PaginationProps) {
         {props.pages.map((pageNumber) => (
           <PaginationLink
             key={pageNumber}
-            ariaLabel={`${pageNumber + 1}`}
+            ariaLabel={`Page ${pageNumber + 1}`}
```

Comment:
other aria-labels aren't configurable, so changing this one to add text isn't a breaking change

Reply:
The tests would fail or need to be skipped between the tests. Happy to revert in this PR if needed

Reply:
Ah that's right, I saw you use those new aria label strings in tests. It's fine for me.